src/components/Checkbox.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from 'react';
+
+import { db } from '../lib/firebase';
+
+const Checkbox = ({ item, shoppingList }) => {
+  const [checked, setChecked] = useState(false);
+
+  const { name, id, lastPurchasedDate } = item;
+
+  const isExpired = (date) => {
+    const expiryDate = item.lastPurchasedDate + 60 * 60 * 24;
+    const now = new Date().getTime();
+    return expiryDate < now;
```

Comment:
Isn't it supposed to be `expiryDate > now`? When I try to load after a few minutes, the items get unchecked?

Reply:
@ciradu2204 , I don't think so -  the logic here is: if expiry date is less than now,  then it's expired. And if it's expired, I reset the checkbox

Reply:
you are right. Thanks

Reply:
hey @katia-utochkina ! so i'm seeing the same thing as cynthia here, where the checked state disappears after a few minutes.

Here's what we're going for with the expected behavior for this one: 

When you 'check' the box, the app's behavior should theoretically keep the box checked for a full 24 hours.

In order to see the effect of that 24 hours pass (without actually waiting 24 hours lol!), you can manipulate the data in Firestore by changing the item's `lastPurchasedOn` value to show that 24 hours have passed (by changing the timestamp).

I think in this case, you want the check to be something like

```
const isExpired = (date) => {
...
return now - lastPurchasedOn > (24 hours in milliseconds)
```